### PR TITLE
AvielCo/issue49

### LINKED ISSUE #49: [BACKEND] Upload image and save in DB
When user is logged in, get an image, save it in lower dimensions, save the image in binary format (base64) inside user's history.

### PR DESCRIPTION
**Pull request by AvielCo**
Fixes #49

1. User can now upload an image to the server (uploading to specific path that belongs to the user with the help of the Access Token)

2. After the user has uploaded an image, the user can send the image to the predict function to recive an output.
the output represent the image details (origin, shape, probability).

### CODE REVIEW

**Comment by AvielCo** [comment]
One more thing:
After the image gets processed and the user gets the output.
The image is resized to 400px width (height is auto) and saved to other save location that belonges to the specific user. (The DB also updates with the image details)

**Comment by AvielCo** [comment]
Please note that the History model has been changed due to the base64 problem.

**Comment by AvielCo** [comment]
And last thing, when we will have the prediction models, we need to uncomment most of the lines in py-files/predict.py

File: backend/py-files/dual_print.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 def dual_print(massage, level="info"):
```
Comment (SoloNoah) [inline backend/py-files/dual_print.py:4]: message? :P

Comment (AvielCo) [reply backend/py-files/dual_print.py:4]: Do you want some massage?
btw its old :( sorry

**Review by SoloNoah — APPROVED** [review]
You DA BOMB (.com because Emilia said to add .com.)

**Comment by AvielCo** [comment]
Spasiva bratan @SoloNoah

File: backend/py-files/dual_print.py (hunk 1 above)
Comment (SoloNoah) [reply backend/py-files/dual_print.py:4]: ohhhh yeaaaa babyy. put some oil.

**Pull request merged**